Makefile

```diff
@@ -101,6 +101,9 @@ test: jsonnet libjsonnet.so libjsonnet_test_snippet libjsonnet_test_file
 	cd test_suite ; ./run_tests.sh
 	cd test_suite ; ./run_fmt_tests.sh
 
+reformat:
```

Comment:
`all` should probably depend on this.

also I think you should declare this as [`.PHONY`](https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html)